logistics/scripts/activities/build/promote-packages.ps1

```diff
@@ -42,6 +45,21 @@ Invoke-Task -name 'Gather package versions' -task {
         (Select-ConfigurationPackages $configurationFile)
     )
 
+    $keysToRemove = @()
+
```

Comment:
This code is remove the unwanted package like here edfi.suite3.ods.standard & edfi.suite3.odsapi.testsdk
$packages = @{
  "edfi.suite3.ods.standard" = "1.0.0"
  "edfi.suite3.ods.standard.4.0.0" = "7.0.12"
  "edfi.suite3.odsapi.testsdk.4.0.0" = "latest"
  "edfi.suite3.odsapi.testsdk" = "7.0.921"
}

Reply:
as per https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/actions/runs/5017187991/jobs/8994999884#step:5:123
 packages are filtered out, ant not published